devmand/gateway/src/devmand/channels/cli/TimeoutTrackingCli.cpp

```diff
@@ -71,9 +71,13 @@ folly::SemiFuture<std::string> TimeoutTrackingCli::executeRead(
 
 folly::SemiFuture<std::string> TimeoutTrackingCli::executeWrite(
     const WriteCommand cmd) {
-  return executeSomething(cmd, "TTCli.executeWrite", [this, cmd]() {
-    return timeoutTrackingParameters->cli->executeWrite(cmd);
-  });
+  return executeSomething(
+             cmd,
+             "TTCli.executeWrite",
+             [this, cmd]() {
```

Comment:
Capturing `this` in lambdas is problematic (in general). Can we capture timeoutTrackingParameters instead?

Reply:
Yes will do... 

This is just a reformat of existing code + I think the lambda is called from a protected context inside executeSomething

Reply:
ok agree.